x-pack/plugins/apm/public/components/app/ServiceDetails/ServiceIntegrations/MachineLearningFlyout/view.tsx

```diff
@@ -73,6 +73,9 @@ export function MachineLearningFlyoutView({
   }
 
   const isLoadingMLJob = status === FETCH_STATUS.LOADING;
+  const mlIsNotAvailable =
+    (status === FETCH_STATUS.SUCCESS && hasMLJob === undefined) ||
+    status === FETCH_STATUS.FAILURE;
```

Comment:
If you re-throw in `getHasMLJob` this can be changed to:
```suggestion
  const isMlAvailable = status === FETCH_STATUS.FAILURE;
```

btw I switched the negation - hope that's ok